### PR TITLE
add namespace injection ignore value

### PR DESCRIPTION
**Pull request by jacob-delgado**
**Please provide a description of this PR:**

An extension to https://istio.io/latest/docs/setup/install/multiple-controlplanes/

Having default revision based control-planes work by ignoring defined namespaces. Users can opt-out of namespaces without having to use revision-based installations.

### CODE REVIEW

File: releasenotes/notes/default-revision-ignores-namespaces.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue: []
+releaseNotes:
+  - |
+    **Added** setting .Values.sidecarInjectiorWebhook.enableNamespacesByDefault for default revision
+    based control-planes so sidecar injection on namesapaces with istio-injection=enabled will be ignored.
```
Comment (dgn) [inline releasenotes/notes/default-revision-ignores-namespaces.yaml:8]: typo: namespaces

**Review by dgn — APPROVED** [review]

File: releasenotes/notes/default-revision-ignores-namespaces.yaml (hunk 1 above)
Comment (jacob-delgado) [inline releasenotes/notes/default-revision-ignores-namespaces.yaml:8]:
```suggestion
    based control-planes so sidecar injection on namespaces with istio-injection=enabled will be ignored.
```

**Comment by howardjohn** [comment]
Sorry I don't fully get it. Why would you have istio-injection=true but not inject?

Are multiple webhooks from different installs looking at istio-injection=true? If so, wouldn't you want an allow-list of namespaces instead of a denylist?

Overall not against it, just want to make sure I understand

**Comment by bleggett** [comment]
Yeah I don't really want to complicate the matrix of what causes injection to happen, it's already a little too baroque.

Is the problem here simply that defaultRevision control planes don't ignore namespaces labeled explicitly for other revisions, and should?

**Comment by linsun** [comment]
@jacob-delgado agree with comments earlier, would be great if you can clarify what scenarios existing injection configuration would not work in your scenario.

**Pull request closed without merging**